approach.tex

```diff
@@ -162,9 +162,9 @@ \subsubsection{Test Documents}
 
 Test Specifications consist of a mixture of material which are directly written and curated in GitHub by the corresponding component owner, and sections which are automatically generated from the contents of Jira.
 
-The \textbf{Test Plan and Report}\footnote{For historical reasons, test plan and report documents use the handle ``DMTR''.}  describes all the information related to a particular test ``campaign'' --- that is, they describe the contents a test plan and the results of executing it.
-Since test plans and reports describe the results of a particular campaign, updates are limited to minor corrections (for example, to spelling).
-A new test campaign, even one which repeated the same test plan, would result in the creation of a new document to describe the new results.
+The \textbf{Test Plan and Report}\footnote{For historical reasons, test plan and report documents use the handle ``DMTR''.}  describes all the information related to a particular test cycle --- that is, they describe the contents of the test cycle and the results of executing it.
```

Comment:
I understand that if  we use the concept "campaign" we have to define itfirst. However, using the word "cycle" instead is misleading.
A "test campaign" includes the entire test activity that is planned using a Jira "Test Plan" and executed using one or more Jira "Test Cycles" (object previously identified as Test Run)

Reply:
directly in your branch?

Reply:
Yes, sounds good.